### PR TITLE
UI updatas: project.views, project show page, #participants responded…

### PR DESCRIPTION
**Pull request by Aakup**
Mostly styling.s Added the progress bar for number of participants responded. Modified the UI of the project main page. Completely redesigned the show page. Edited cucumber tests.

### CODE REVIEW

**Review by kev-chien — APPROVED** [review]
Thanks for making the user interface nicer! Everything looks good.

File: app/views/projects/index.html.erb [hunk 1]
```diff
@@ -35,12 +35,29 @@
   width:100%;
   height:100%;
   top:0;
-  left: 0;}
+  left: 0;
+  transition: 0.5s;}
+  span:hover {background-color: #42bff4;
```
Comment (lawrencechen14) [inline app/views/projects/index.html.erb:40]: Please put this CSS in a different file!

File: app/views/projects/index.html.erb [hunk 2]
```diff
@@ -55,11 +72,24 @@
...
-              <a href= <%=project_path(project.id)%> <%= project.project_name %> style="text-decoration: none"><span class="proj"></span>
+              <a href= <%=project_path(project.id)%> <%= project.project_name %> style="text-decoration: none"><span1 class="proj1"></span1>
                 <h3 style="color: black"> <%= project.project_name%></h3>
               </a>
-              <h11> Project ID: <%= project.id %></h11>
+              <div class="progress">
+
+                <% if responded(project.id)*100 > 75 %>
+                  <div class="progress-bar bg-success" role="progressbar" style= "width: <%=responded(project.id)*100%>%" aria-valuenow= "<%=responded(project.id)*100%>" aria-valuemin="0" aria-valuemax="100"></div>
+                <% elsif responded(project.id)*100 > 50 %>
```
Comment (lawrencechen14) [inline app/views/projects/index.html.erb:84]: Please no inline CSS, put it in a different stylesheet!

File: app/views/projects/show.html.erb [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
+    <% else %>
+      <p><strong>Current Timeslot Duration(miniutes):</strong> Not set yet</p>
+    <% end %>
+  </div>
+
+  <div class="project-times-table">
+    <div class="project-times-dateentry list-group-item list-group-item-primary">
+      <strong>
+        Participant Email
+      </strong>
```
Comment (lawrencechen14) [inline app/views/projects/show.html.erb:19]: Great job with this one, it looks really great now!

File: app/controllers/projects_controller.rb [hunk 4]
```diff
@@ -87,4 +88,15 @@ def set_project
   def project_params
     params.require(:project).permit(:project_name)
   end
+
+  def responded(id)
+    count = 0
+    Participant.where(project_id: id).pluck(:last_responded).each do |answer|
```
Comment (lawrencechen14) [inline app/controllers/projects_controller.rb:94]: Nice way to get the number of people who responded here! Maybe clarify that this method returns a fraction of the people.

**Review by lawrencechen14 — APPROVED** [review]

**Pull request merged**